release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -680,17 +680,6 @@ In the following tables, features are marked with the following statuses:
 
 {product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for earlier setups. The `REGISTRY_OPENSHIFT_SERVER_ADDR` variable can be used in its place.
 
-.APIs removed from Kubernetes 1.27
```

Comment:
I know there are none for 1.28, but does it make sense to preserve this table (commented out and empty) for a future release so it's consistent with past versions?

Reply:
You are right, that's the better way to go. I'll change it.

Reply:
@jeana-redhat Changed it to be commented out. Ready for merge.

Reply:
will do, whenever the build is done :)